ydb/docs/en/core/integrations/liquibase.md

```diff
@@ -0,0 +1,273 @@
+# Support for the YDB dialect in the Liquibase migration tool. 
+
+## Introduction 
+
+In today's world of software development, database schema versioning is a critical task to ensure consistency and the ability to roll back changes. Database migration tools, such as Liquibase, provide tools for versioning, tracking, and applying changes to the schema. 
+
+However, the emergence of new database management systems (DBMS) has made it necessary to integrate with these systems in order to support their unique capabilities and SQL dialects. 
+
+We will discuss the results of integrating DB dialect with Liquibase in this article.
```

Comment:
```suggestion
```
this might be ok as an introduction to a blog post, but looks irrelevant for documentation